Tests/BasicsTests/CancellatorTests.swift

```diff
@@ -391,6 +392,7 @@ fileprivate struct Worker {
         return self.semaphore.wait(timeout: deadline)
     }
 
+    @Sendable
```

Comment:
I wonder if we should try enabling InferSendableFromCaptures instead of these explicit attributes in few places including properties.

Reply:
cc @MaxDesiatov, we can do that without experimental flags too.

Reply:
How would you do that without experimental flags? Does Swift 6 language mode include support for that?

Reply:
It’s an upcoming feature flag.